pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -1309,7 +1291,15 @@ func doNetworkInstanceDelete(
 		}
 		DNSStopMonitor(status.BridgeNum)
 	}
+	if status.BridgeMac != "" {
+		mac, err := net.ParseMAC(status.BridgeMac)
+		if err != nil {
+			log.Fatal("ParseMAC failed: ", status.BridgeMac, err)
+		}
+		releaseIPv4FromNetworkInstance(ctx, status, mac)
+	}
 	networkInstanceBridgeDelete(ctx, status)
+	appNumOnUNetBaseDelete(status.UUID)
```

Comment:
Why are you doing this delete twice - here and in the caller?
Makes sense to
1. just to it in the caller
2. make appNumOnUNetBaseDelete fatal if the entry does not exist in the map i.e., Get() fails.